doc/build-openbsd.md

```diff
@@ -46,15 +44,16 @@ from ports. However you can build it yourself, [using depends](/depends).
 Refer to [depends/README.md](/depends/README.md) for detailed instructions.
 
 ```bash
+pkg_add gmake gtar
```

Comment:
bash is being removed above, but if this is going to be added (probably shouldn't duplicate depends here), then it should probably also be here.

Reply:
Why? But bash is not needed to build depends.

Reply:
> Why? bash is not needed to build depends.

Yes it is: https://github.com/bitcoin/bitcoin/blob/master/depends/gen_id

Reply:
Oops. You're right.

But why depends do not fail without bash being installed?

Reply:
Because generating the ID is currently best-effort, and trying to invoke `bash` even when it's not installed, will produce some output.

Reply:
I confirm:

- without `bash`:
```
$ gmake print-boost_build_id_long 
env: bash: No such file or directory
env: bash: No such file or directory
boost_build_id_long=boost-1.81.0-fc95b2d08969bfab552a815064aa6a30cc97b2b2c7d7a3cf9dc7d4f67781b91e-release  
$ gmake print-final_build_id      
env: bash: No such file or directory
env: bash: No such file or directory
final_build_id=8e0fd0c7a8b
```

- with `bash`:
```
$ gmake print-boost_build_id_long 
boost_build_id_long=boost-1.81.0-fc95b2d08969bfab552a815064aa6a30cc97b2b2c7d7a3cf9dc7d4f67781b91e-release  ca1ce530b13a4de03966f87414b24766b48463b14e6b91165c264c7f21058e09
$ gmake print-final_build_id      
final_build_id=5cba29525e3
```

Reply:
> bash is being removed above, but if this is going to be added (probably shouldn't duplicate depends here), then it should probably also be here.

Added.